Dockerfile

```diff
@@ -113,6 +115,8 @@ VOLUME /src/MLOS/target
 VOLUME /src/MLOS/tools
 VOLUME /src/MLOS/temp
 
+VOLUME /src/MLOS/external/ExternalIntegrationExample/build
```

Comment:
These aren't strictly necessary, but they make testing on multiple docker instances at the same time from a single source tree possible.